charts/kube-state-metrics/templates/deployment.yaml

```diff
@@ -147,17 +147,41 @@ spec:
         {{- end }}
         {{- end }}
         livenessProbe:
+          failureThreshold: {{ .Values.livenessProbe.failureThreshold }}
           httpGet:
+            {{- if .Values.kubeRBACProxy.enabled }}
```

Comment:
Maybe this should be configurable too, with a comment for RBAC proxy users in the values file?